src/generated/templateImports.ts

```diff
@@ -0,0 +1,8 @@
+export default [
```

Comment:
This is a generated file from [here](https://github.com/wagmi-dev/create-wagmi/pull/18/files#diff-8fed899bdbc24789a7bb4973574e624ed6207c6ce572338bc3c3e117672b2a20R11).